From 48c537b4ba35b7775f6b5380ff4d5b8e41c96b17 Mon Sep 17 00:00:00 2001
From: Your Name <you@example.com>
Date: Wed, 8 Jul 2020 07:24:58 +0000
Subject: [PATCH] change port

---
 app.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/app.py b/app.py
index d3c9203..1efd75a 100644
--- a/app.py
+++ b/app.py
@@ -11,4 +11,4 @@ def hello_world():
 
 
 if __name__ == '__main__':
-    app.run(host='0.0.0.0')
+    app.run(host='0.0.0.0', port=7777)
-- 
2.17.1

